pom.xml

```diff
@@ -878,9 +878,9 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>2.17</version>
+                <version>3.0.0</version>
                 <configuration>
-                    <sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
+                    <sourceDirectories>${project.build.sourceDirectory}</sourceDirectories>
```

Comment:
@leventov would you tell me why this change is needed? Apparently, `sourceDirectories` needs a list which looks related to https://github.com/apache/incubator-druid/issues/6668.

Reply:
https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#sourceDirectory

Reply:
Thanks.